optuna/visualization/_param_importances.py

```diff
@@ -24,6 +25,64 @@
 logger = get_logger(__name__)
 
 
+class _ImportancesInfo(NamedTuple):
+    title: str
+    x_axis_name: str
+    y_axis_name: str
```

Comment:
Similarly to `plot_edf`, I think those are simple enough to be passed directly as argument to function that creates figure in each of backends. No need to store as part of info class.

Reply:
Thank you for yoru coments. I agree with you; I will remove these values from the named tuple.